src/bin/pg_autoctl/ini_file.c

```diff
@@ -287,6 +287,11 @@ ini_option_to_string(IniOption *option, char *dest, size_t size)
 	{
 		case INI_STRING_T:
 		{
+			if (option->strValue == NULL || *(option->strValue) == NULL)
```

Comment:
not very trivial to understand. we actually mean is null or empty string. Perhaps consider a macro

Reply:
Well we have a `char **` so that's two pointers, any of those could be NULL. I added a comment to that effect.